File: src/neetcode/Java/_27RemoveElement.java
package neetcode.Java;

public class _27RemoveElement {
    public int removeElement(int[] nums, int val) {
        int k = 0;
        for (int i : nums) {
            if (i == val) {
                nums[k++] = i;
            }
        }
        return k;
    }

    public static void main(String[] args) {
        _27RemoveElement obj = new _27RemoveElement();
        int[] num = {3, 2, 2, 3};
        int val = 3;
        System.out.println(obj.removeElement(num, val));
    }
}

File: src/neetcode/kotlin/_27RemoveElement.kt
package neetcode.kotlin

class _27RemoveElement {
    fun removeElement(nums: IntArray, `val`: Int): Int {
        var k = 0
        for (i in nums) {
            if (i == `val`) {
                nums[k++] = i
            }
        }
        return k
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val obj = _27RemoveElement()
            val num = intArrayOf(3, 2, 2, 3)
            val `val` = 3
            println(obj.removeElement(num, `val`))
        }
    }
}
